frontend/src/components/Json.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+//import Data from './data.json';
+
+export class Json extends Component {
+    constructor(props) {
+      super(props)
+    
+      this.state = {
+         
+      }
+    }
+    
+  render() {
+    
+    return (
+      <div>
+        
+      </div>
+    );
+  }
+}
+
```

Comment:
This componenet seems to be empty... and not use ... are you done working on it or it's not necessary ? 🤔

Reply:
It's not necessary. I'll delete it